src/hindsight.c

```diff
@@ -108,7 +108,7 @@ int main(int argc, char* argv[])
       break; // shutting down
     }
     hs_write_checkpoints(&cpw, &cfg.cp_reader);
-    if (cfg.load_path[0] != 0 && ++cnt == 59) { // scan just before emitting the stats
+    if (cfg.load_path[0] != 0 && ++cnt == cfg.load_interval) { // scan just before emitting the stats
```

Comment:
This no longer scans before emitting stats (since they are flushed once a minute)